app/controllers/cws_controller.rb

```diff
@@ -1,10 +1,19 @@
 class CwsController < ApplicationController
-  def new; end
+  def new
+    @form = CwSelection.new
+    @estimate = EstimateModel.from_session(session_data)
+  end
 
   def create
-    send_data File.open(Rails.root.join("/lib/cw1-form.pdf")),
-              filename: "cw1-form.pdf",
-              type: "application/pdf"
+    @form = CwSelection.new(params[:cw_selection]&.permit(:form_type))
```

Comment:
So yeah, at the moment on line 10 we just send the same pdf, no matter what `form_type` is in the `params` here. In future this method will have some sort of service call where it passes in the session_data and the chosen form type and it receives back a file that it can then pass to `send_data`. But that's outside the scope of this ticket.